### ISSUE TITLE
Crash when right clicking tar distiller with bucket

### ISSUE THREAD

**Issue by alandoherty** [Flenix/FlenixRoads#31]
The crashlog:

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  XaeroMinimapPlugin (Xaeros_Minimap_20.20.1_Forge_1.12.jar)
  TransformerLoader (OpenComputers-MC1.12.2-1.7.5.192.jar)
  MekanismCoremod (Mekanism-1.12.2-9.8.3.390.jar)
  MicdoodlePlugin (MicdoodleCore-1.12.2-4.0.2.280.jar)
  XaeroWorldMapPlugin (XaerosWorldMap_1.10.1_Forge_1.12.jar)
  MalisisCorePlugin (malisiscore-1.12.2-6.5.1.jar)
  SoundUnpack (OpenSecurity-1.12.2-1.0-46.jar)
  ObfuscatePlugin (obfuscate-0.4.2-1.12.2.jar)
  CTMCorePlugin (CTM-MC1.12.2-1.0.2.31.jar)
Contact their authors BEFORE contacting forge

// My bad.

Time: 8/26/20 9:45 PM
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.minecraftforge.fluids.FluidUtil.getFilledBucket(FluidUtil.java:725)
	at com.silvaniastudios.roads.blocks.tileentities.distiller.TarDistillerBlock.func_180639_a(TarDistillerBlock.java:73)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187099_a(PlayerControllerMP.java:423)
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1579)
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2254)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:2020)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1808)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1098)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.minecraftforge.fluids.FluidUtil.getFilledBucket(FluidUtil.java:725)
	at com.silvaniastudios.roads.blocks.tileentities.distiller.TarDistillerBlock.func_180639_a(TarDistillerBlock.java:73)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187099_a(PlayerControllerMP.java:423)
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1579)
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2254)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:2020)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [GCEntityClientPlayerMP['holon3'/483520, l='MpServer', x=-436.30, y=73.00, z=157.05]]
	Chunk stats: MultiplayerChunkCache: 441, 441
	Level seed: 0
	Level generator: ID 06 - BIOMESOP, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-150,64,322), Chunk: (at 10,4,2 in -10,20; contains blocks -160,0,320 to -145,255,335), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 5226290 game time, 3875099 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 83 total; [EntityCreeper['Creeper'/488452, l='MpServer', x=-420.89, y=46.00, z=148.54], EntityCreeper['Creeper'/487427, l='MpServer', x=-373.50, y=60.00, z=160.50], EntityCreeper['Creeper'/488971, l='MpServer', x=-419.50, y=25.00, z=116.50], EntitySkeleton['Skeleton'/488970, l='MpServer', x=-418.50, y=25.00, z=118.50], EntitySkeleton['Skeleton'/488969, l='MpServer', x=-413.50, y=25.00, z=117.50], EntitySkeleton['Skeleton'/488968, l='MpServer', x=-414.50, y=25.00, z=118.50], EntityZombie['Zombie'/489239, l='MpServer', x=-484.50, y=41.00, z=209.50], EntityLlama['Llama'/483613, l='MpServer', x=-491.90, y=108.00, z=167.50], EntityLlama['Llama'/483612, l='MpServer', x=-485.89, y=110.00, z=176.46], EntityBOPBoat['Boat'/483611, l='MpServer', x=-488.69, y=129.52, z=152.45], EntityLlama['Llama'/483610, l='MpServer', x=-492.90, y=94.00, z=169.58], EntityLlama['Llama'/483609, l='MpServer', x=-498.38, y=91.00, z=171.44], EntityBat['Bat'/490021, l='MpServer', x=-384.50, y=52.00, z=133.50], EntityBat['Bat'/490020, l='MpServer', x=-381.50, y=52.00, z=132.50], EntityBat['Bat'/489259, l='MpServer', x=-374.80, y=50.87, z=152.54], EntityZombie['Zombie'/487735, l='MpServer', x=-479.30, y=21.00, z=209.70], EntityZombieVillager['Zombie Villager'/487733, l='MpServer', x=-412.50, y=41.00, z=126.50], EntityZombieVillager['Zombie Villager'/487732, l='MpServer', x=-407.50, y=41.00, z=127.50], EntityCreeper['Creeper'/490054, l='MpServer', x=-377.50, y=49.00, z=144.50], EntityZombie['Zombie'/489300, l='MpServer', x=-388.50, y=56.00, z=148.50], EntityOffRoader['Off Roader'/483667, l='MpServer', x=-413.72, y=73.00, z=160.15], EntityBat['Bat'/488543, l='MpServer', x=-387.22, y=16.08, z=125.75], EntityBat['Bat'/488542, l='MpServer', x=-395.30, y=18.35, z=114.33], EntityBat['Bat'/489053, l='MpServer', x=-385.44, y=37.00, z=183.11], EntityBat['Bat'/489052, l='MpServer', x=-367.73, y=35.07, z=181.79], EntityCreeper['Creeper'/489313, l='MpServer', x=-407.50, y=12.00, z=169.50], EntityLlama['Llama'/483693, l='MpServer', x=-428.50, y=139.00, z=101.08], EntitySkeleton['Skeleton'/489079, l='MpServer', x=-400.50, y=56.00, z=151.50], EntityRabbit['Rabbit'/479349, l='MpServer', x=-362.12, y=69.00, z=110.50], EntityEnderman['Enderman'/488562, l='MpServer', x=-367.50, y=11.00, z=161.50], EntityCreeper['Creeper'/488561, l='MpServer', x=-363.50, y=11.00, z=164.50], EntitySkeleton['Skeleton'/489087, l='MpServer', x=-434.50, y=19.00, z=111.50], EntityCreeper['Creeper'/489854, l='MpServer', x=-409.50, y=42.00, z=125.50], EntityCreeper['Creeper'/489853, l='MpServer', x=-406.50, y=42.00, z=126.50], EntityBat['Bat'/489336, l='MpServer', x=-367.75, y=25.52, z=204.75], EntityLlama['Llama'/483719, l='MpServer', x=-386.55, y=74.00, z=122.09], EntitySkeleton['Skeleton'/489092, l='MpServer', x=-371.50, y=18.00, z=89.50], EntitySkeleton['Skeleton'/487552, l='MpServer', x=-441.50, y=25.00, z=82.50], EntityZombie['Zombie'/488331, l='MpServer', x=-438.50, y=38.00, z=120.50], EntityLlama['Llama'/483721, l='MpServer', x=-392.54, y=72.00, z=114.89], EntityLlama['Llama'/483720, l='MpServer', x=-378.92, y=77.00, z=119.77], EntitySkeleton['Skeleton'/488854, l='MpServer', x=-375.50, y=60.00, z=164.50], EntitySkeleton['Skeleton'/489878, l='MpServer', x=-412.50, y=21.00, z=191.50], EntitySkeleton['Skeleton'/485269, l='MpServer', x=-367.50, y=52.00, z=144.50], EntityZombie['Zombie'/488853, l='MpServer', x=-374.50, y=60.00, z=165.50], EntityCreeper['Creeper'/489109, l='MpServer', x=-412.50, y=22.00, z=193.50], EntitySkeleton['Skeleton'/489877, l='MpServer', x=-412.50, y=21.00, z=190.50], EntityZombie['Zombie'/488852, l='MpServer', x=-375.50, y=60.00, z=167.50], EntityZombie['Zombie'/488851, l='MpServer', x=-378.50, y=60.00, z=167.50], EntitySquid['Squid'/489630, l='MpServer', x=-397.54, y=52.17, z=203.28], EntitySquid['Squid'/489629, l='MpServer', x=-397.25, y=52.24, z=197.14], EntitySquid['Squid'/489628, l='MpServer', x=-405.23, y=48.54, z=199.96], EntitySquid['Squid'/489627, l='MpServer', x=-408.67, y=51.70, z=190.60], EntityCreeper['Creeper'/488602, l='MpServer', x=-400.50, y=15.00, z=173.50], EntityZombie['Zombie'/488601, l='MpServer', x=-400.50, y=15.00, z=174.50], EntitySkeleton['Skeleton'/489881, l='MpServer', x=-411.50, y=14.00, z=101.50], EntitySkeleton['Skeleton'/489880, l='MpServer', x=-410.50, y=14.00, z=101.50], EntityBat['Bat'/482722, l='MpServer', x=-421.25, y=50.10, z=146.60], EntitySquid['Squid'/486575, l='MpServer', x=-481.42, y=61.82, z=215.97], EntityZombie['Zombie'/488108, l='MpServer', x=-404.50, y=21.00, z=116.50], EntityBat['Bat'/489386, l='MpServer', x=-414.65, y=42.81, z=120.62], EntityBat['Bat'/489385, l='MpServer', x=-403.39, y=44.00, z=128.49], EntitySquid['Squid'/489654, l='MpServer', x=-470.60, y=57.39, z=220.82], EntitySquid['Squid'/489652, l='MpServer', x=-466.94, y=55.14, z=214.54], EntitySquid['Squid'/489651, l='MpServer', x=-478.81, y=57.64, z=214.88], EntitySquid['Squid'/489650, l='MpServer', x=-475.79, y=55.20, z=210.98], EntitySquid['Squid'/486589, l='MpServer', x=-406.71, y=59.77, z=122.79], EntitySquid['Squid'/486588, l='MpServer', x=-413.00, y=60.15, z=125.58], EntityStorageTrailer['Storage Trailer'/414138, l='MpServer', x=-423.13, y=73.00, z=159.89], EntityTractor['Tractor'/438715, l='MpServer', x=-420.50, y=73.00, z=159.92], EntityStorageTrailer['Storage Trailer'/414137, l='MpServer', x=-424.98, y=73.00, z=159.88], EntitySkeleton['Skeleton'/485059, l='MpServer', x=-393.28, y=57.00, z=145.49], EntityZombie['Zombie'/488909, l='MpServer', x=-373.50, y=17.00, z=182.50], EntityCreeper['Creeper'/486345, l='MpServer', x=-390.50, y=49.00, z=144.50], EntitySkeleton['Skeleton'/488912, l='MpServer', x=-370.66, y=18.00, z=184.49], EntityCreeper['Creeper'/485595, l='MpServer', x=-373.84, y=60.00, z=163.17], EntitySkeleton['Skeleton'/488408, l='MpServer', x=-375.50, y=38.00, z=154.50], EntityBat['Bat'/486628, l='MpServer', x=-379.25, y=49.10, z=160.25], EntityZombie['Zombie'/489185, l='MpServer', x=-503.50, y=14.00, z=95.50], EntityCreeper['Creeper'/489184, l='MpServer', x=-505.79, y=13.00, z=101.47], GCEntityClientPlayerMP['holon3'/483520, l='MpServer', x=-436.30, y=73.00, z=157.05], EntitySkeleton['Skeleton'/489972, l='MpServer', x=-378.50, y=18.00, z=169.50], EntitySpider['Spider'/489971, l='MpServer', x=-379.50, y=18.00, z=168.50]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:420)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2741)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:427)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_261, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1604907032 bytes (1530 MB) / 5962727424 bytes (5686 MB) up to 7635730432 bytes (7282 MB)
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx8192m
	IntCache: cache: 0, tcache: 0, allocated: 5, tallocated: 104
	FML: MCP 9.42 Powered by Forge 14.23.5.2854 72 mods loaded, 72 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State  | ID                             | Version               | Source                                              | Signature                                |
	|:------ |:------------------------------ |:--------------------- |:--------------------------------------------------- |:---------------------------------------- |
	| LCHIJA | minecraft                      | 1.12.2                | minecraft.jar                                       | None                                     |
	| LCHIJA | mcp                            | 9.42                  | minecraft.jar                                       | None                                     |
	| LCHIJA | FML                            | 8.0.99.99             | forge-1.12.2-14.23.5.2854-universal.jar             | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCHIJA | forge                          | 14.23.5.2854          | forge-1.12.2-14.23.5.2854-universal.jar             | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCHIJA | micdoodlecore                  |                       | minecraft.jar                                       | None                                     |
	| LCHIJA | xaerominimap_core              | 1.12.2-1.0            | minecraft.jar                                       | None                                     |
	| LCHIJA | xaeroworldmap_core             | 1.12.2-1.0            | minecraft.jar                                       | None                                     |
	| LCHIJA | obfuscate                      | 0.4.2                 | minecraft.jar                                       | None                                     |
	| LCHIJA | opencomputers|core             | 1.7.5.192             | minecraft.jar                                       | None                                     |
	| LCHIJA | biomesoplenty                  | 7.0.1.2444            | BiomesOPlenty-1.12.2-7.0.1.2444-universal.jar       | None                                     |
	| LCHIJA | buildcraftlib                  | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | buildcraftcore                 | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | buildcraftbuilders             | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | buildcrafttransport            | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | buildcraftsilicon              | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | crafttweaker                   | 4.1.20                | CraftTweaker2-1.12-4.1.20.585.jar                   | None                                     |
	| LCHIJA | buildcraftcompat               | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | buildcraftenergy               | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | buildcraftfactory              | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | buildcraftrobotics             | 7.99.24.6             | buildcraft-all-7.99.24.6.jar                        | None                                     |
	| LCHIJA | chameleon                      | 1.12-4.1.3            | Chameleon-1.12-4.1.3.jar                            | None                                     |
	| LCHIJA | ctm                            | MC1.12.2-1.0.2.31     | CTM-MC1.12.2-1.0.2.31.jar                           | None                                     |
	| LCHIJA | jei                            | 4.15.0.296            | jei_1.12.2-4.15.0.296.jar                           | None                                     |
	| LCHIJA | chisel                         | MC1.12.2-1.0.2.45     | Chisel-MC1.12.2-1.0.2.45.jar                        | None                                     |
	| LCHIJA | codechickenlib                 | 3.2.3.358             | CodeChickenLib-1.12.2-3.2.3.358-universal.jar       | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| LCHIJA | ctgui                          | 1.0.0                 | CraftTweaker2-1.12-4.1.20.585.jar                   | None                                     |
	| LCHIJA | crafttweakerjei                | 2.0.3                 | CraftTweaker2-1.12-4.1.20.585.jar                   | None                                     |
	| LCHIJA | energyconverters               | 1.3.7.30              | energyconverters_1.12.2-1.3.7.30.jar                | None                                     |
	| LCHIJA | forgemultipartcbe              | 2.6.2.83              | ForgeMultipart-1.12.2-2.6.2.83-universal.jar        | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| LCHIJA | microblockcbe                  | 2.6.2.83              | ForgeMultipart-1.12.2-2.6.2.83-universal.jar        | None                                     |
	| LCHIJA | minecraftmultipartcbe          | 2.6.2.83              | ForgeMultipart-1.12.2-2.6.2.83-universal.jar        | None                                     |
	| LCHIJA | furenikusroads                 | 1.1.7                 | Fureniku's+Roads-1.1.7.jar                          | None                                     |
	| LCHIJA | cfm                            | 6.3.1                 | furniture-6.3.1-1.12.2.jar                          | None                                     |
	| LCHIJA | galacticraftcore               | 4.0.2.280             | GalacticraftCore-1.12.2-4.0.2.280.jar               | None                                     |
	| LCHIJA | galacticraftplanets            | 4.0.2.280             | Galacticraft-Planets-1.12.2-4.0.2.280.jar           | None                                     |
	| LCHIJA | gravestone                     | 1.10.3                | gravestone-1.10.3.jar                               | None                                     |
	| LCHIJA | cgm                            | 0.15.3                | guns-0.15.3-1.12.2.jar                              | None                                     |
	| LCHIJA | hpm                            | §aMC 1.12.2           | Harry+Potter+Mod+-+1.2.jar                          | None                                     |
	| LCHIJA | icbmclassic                    | 1.12.2-3.3.0.63       | ICBM-classic-1.12.2-3.3.0b63.jar                    | None                                     |
	| LCHIJA | ironchest                      | 1.12.2-7.0.67.844     | ironchest-1.12.2-7.0.72.847.jar                     | None                                     |
	| LCHIJA | malisiscore                    | 1.12.2-6.5.1-SNAPSHOT | malisiscore-1.12.2-6.5.1.jar                        | None                                     |
	| LCHIJA | malisisdoors                   | 1.12.2-7.3.0          | malisisdoors-1.12.2-7.3.0.jar                       | None                                     |
	| LCHIJA | mcwbridges                     | 1.0.4                 | mcw-bridges-1.0.4-mc1.12.2.jar                      | None                                     |
	| LCHIJA | mekanism                       | 1.12.2-9.8.3.390      | Mekanism-1.12.2-9.8.3.390.jar                       | None                                     |
	| LCHIJA | mekanismgenerators             | 1.12.2-9.8.3.390      | MekanismGenerators-1.12.2-9.8.3.390.jar             | None                                     |
	| LCHIJA | mekanismtools                  | 1.12.2-9.8.3.390      | MekanismTools-1.12.2-9.8.3.390.jar                  | None                                     |
	| LCHIJA | mousetweaks                    | 2.10                  | MouseTweaks-2.10-mc1.12.2.jar                       | None                                     |
	| LCHIJA | mrtjpcore                      | 2.1.4.43              | MrTJPCore-1.12.2-2.1.4.43-universal.jar             | None                                     |
	| LCHIJA | nuclearcraft                   | 2.18w                 | NuclearCraft-2.18w-1.12.2.jar                       | None                                     |
	| LCHIJA | opencomputers                  | 1.7.5.192             | OpenComputers-MC1.12.2-1.7.5.192.jar                | None                                     |
	| LCHIJA | omlib                          | 3.1.4-249             | omlib-1.12.2-3.1.4-249.jar                          | None                                     |
	| LCHIJA | eternalsoap.icbm.opencomputers | 1.0                   | OpenComputersICBMAddon-MC1.12.2-1.1.jar             | None                                     |
	| LCHIJA | openglider                     | @VERSION@             | OpenGlider-1.12.1-1.1.0.jar                         | None                                     |
	| LCHIJA | openmodularturrets             | 3.1.11-372            | openmodularturrets-1.12.2-3.1.11-372.jar            | None                                     |
	| LCHIJA | openprinter                    | 0.1.0.9               | OpenPrinter-1.12.2-0.1.0.9.jar                      | None                                     |
	| LCHIJA | opensecurity                   | 1.0-46                | OpenSecurity-1.12.2-1.0-46.jar                      | None                                     |
	| LCHIJA | harvestcraft                   | 1.12.2zb              | Pam's+HarvestCraft+1.12.2zg.jar                     | None                                     |
	| LCHIJA | pneumaticcraft                 | 1.12.2-0.11.15-398    | pneumaticcraft-repressurized-1.12.2-0.11.15-398.jar | None                                     |
	| LCHIJA | projectred-core                | 4.9.4.120             | ProjectRed-1.12.2-4.9.4.120-Base.jar                | None                                     |
	| LCHIJA | projectred-integration         | 4.9.4.120             | ProjectRed-1.12.2-4.9.4.120-integration.jar         | None                                     |
	| LCHIJA | projectred-transmission        | 4.9.4.120             | ProjectRed-1.12.2-4.9.4.120-integration.jar         | None                                     |
	| LCHIJA | projectred-fabrication         | 4.9.4.120             | ProjectRed-1.12.2-4.9.4.120-fabrication.jar         | None                                     |
	| LCHIJA | projectred-illumination        | 4.9.4.120             | ProjectRed-1.12.2-4.9.4.120-lighting.jar            | None                                     |
	| LCHIJA | railcraft                      | 12.0.0                | railcraft-12.0.0.jar                                | a0c255ac501b2749537d5824bb0f0588bf0320fa |
	| LCHIJA | rsgauges                       | 1.2.4                 | rsgauges-1.12.2-1.2.4.jar                           | ed58ed655893ced6280650866985abcae2bf7559 |
	| LCHIJA | simplycats                     | 1.12.2-0.0.3.1        | simplycats-1.12.2-0.0.3.1.jar                       | None                                     |
	| LCHIJA | storagedrawers                 | 5.2.2                 | StorageDrawers-1.12.2-5.4.2.jar                     | None                                     |
	| LCHIJA | treechopper                    | 1.2.4                 | TreeChopper-1.12.2-1.2.4.jar                        | None                                     |
	| LCHIJA | vehicle                        | 0.44.1                | vehicle-mod-0.44.1-alan-1.12.2.jar                  | None                                     |
	| LCHIJA | wearablebackpacks              | 3.1.4                 | WearableBackpacks-1.12.2-3.1.4.jar                  | None                                     |
	| LCHIJA | xaerominimap                   | 20.20.1               | Xaeros_Minimap_20.20.1_Forge_1.12.jar               | None                                     |
	| LCHIJA | xaeroworldmap                  | 1.10.1                | XaerosWorldMap_1.10.1_Forge_1.12.jar                | None                                     |

	Loaded coremods (and transformers): 
XaeroMinimapPlugin (Xaeros_Minimap_20.20.1_Forge_1.12.jar)
  xaero.common.core.transformer.ChunkTransformer
  xaero.common.core.transformer.NetHandlerPlayClientTransformer
  xaero.common.core.transformer.EntityPlayerTransformer
  xaero.common.core.transformer.AbstractClientPlayerTransformer
  xaero.common.core.transformer.WorldClientTransformer
  xaero.common.core.transformer.EntityPlayerSPTransformer
  xaero.common.core.transformer.PlayerListTransformer
TransformerLoader (OpenComputers-MC1.12.2-1.7.5.192.jar)
  li.cil.oc.common.asm.ClassTransformer
MekanismCoremod (Mekanism-1.12.2-9.8.3.390.jar)
  mekanism.coremod.KeybindingMigrationHelper
MicdoodlePlugin (MicdoodleCore-1.12.2-4.0.2.280.jar)
  micdoodle8.mods.miccore.MicdoodleTransformer
XaeroWorldMapPlugin (XaerosWorldMap_1.10.1_Forge_1.12.jar)
  xaero.map.core.transformer.ChunkTransformer
  xaero.map.core.transformer.NetHandlerPlayClientTransformer
  xaero.map.core.transformer.EntityPlayerTransformer
  xaero.map.core.transformer.AbstractClientPlayerTransformer
  xaero.map.core.transformer.WorldClientTransformer
  xaero.map.core.transformer.PlayerListTransformer
MalisisCorePlugin (malisiscore-1.12.2-6.5.1.jar)
  
SoundUnpack (OpenSecurity-1.12.2-1.0-46.jar)
  
ObfuscatePlugin (obfuscate-0.4.2-1.12.2.jar)
  com.mrcrayfish.obfuscate.asm.ObfuscateTransformer
CTMCorePlugin (CTM-MC1.12.2-1.0.2.31.jar)
  team.chisel.ctm.client.asm.CTMTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 441.41' Renderer: 'GeForce GTX 970/PCIe/SSE2'
	Launched Version: MultiMC5
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 970/PCIe/SSE2 GL version 4.6.0 NVIDIA 441.41, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 12x Intel(R) Core(TM) i7-5930K CPU @ 3.50GHz
```

**Comment by Fureniku** [comment]
I couldn't seem to reproduce this, there might've been additional things at play that I needed (what fluids were in the machine and how much of each, and was it just a vanilla, empty bucket?)

However I can see one thing that might cause the crash, so I've put a tentative fix in for 1.1.8. Can you let me know if you still get the crash once that's released (later today or tomorrow most likely)

Thanks

**Comment by Fureniku** [comment]
Couldn't reproduce and no further info, assumed no longer relevant. Please re-open if you're still experiencing the issue and have more information to contribute.
